src/server/services/organisations/organisationSurveys.js

```diff
@@ -444,6 +444,37 @@ const deleteOrganisationSurvey = async feedbackTargetId => {
   }
 }
 
+const getStudentNumbersFromCourseIds = async courseIds => {
+  try {
+    if (!courseIds || courseIds.length < 1) return []
```

Comment:
Small, but I think it's better to check if courseIds.length === 0 istead of < 1. Easier to read